base/uk.ac.stfc.isis.ibex.ui.widgets/src/uk/ac/stfc/isis/ibex/ui/widgets/HelpButton.java

```diff
@@ -0,0 +1,67 @@
+package uk.ac.stfc.isis.ibex.ui.widgets;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.ui.PartInitException;
+import org.eclipse.ui.PlatformUI;
+import org.eclipse.wb.swt.ResourceManager;
+
+import uk.ac.stfc.isis.ibex.logger.IsisLog;
+import uk.ac.stfc.isis.ibex.logger.LoggerUtils;
+
+/**
+ * Universal 'help' button - links to appropriate wiki page.
+ */
+public class HelpButton {
+	
+	private static final String SYMBOLIC_PATH = "uk.ac.stfc.isis.ibex.ui.widgets";
+	private static final String HELP_ICON2 = "/icons/helpIcon.png";
+	private static final String TOOLTIP_TEXT = "Open user manual link in browser for help with '%s': \n%s";
+	
+	private final String tooltipDesc; 
+	@SuppressWarnings("unused")
+	private final String wikiLink;
```

Comment:
Same question for these, if we are only using them once do we need them as members?